test/abstractarray.jl

```diff
@@ -775,3 +775,14 @@ end
 #to_shape
 @test Base.to_shape(()) === ()
 @test Base.to_shape(1) === 1
+
+#vec for multiple arrays
```

Comment:
I think someone has an open PR for the rest of the file, so only need to worry about your addition. What you have there should work, yeah.